### PR TITLE
Open docs in new window

### LINKED ISSUE #79: documentation links in navbar open in same tab
when quarto 1.3 releases add `target: _blank` in navbar option for documentation in the `_quarto.yaml` file. If documentation for v2 is not finished. see quarto-dev/quarto-cli#2284

```yaml
menu:
  - text: OpenProblems v1
    href: https://github.com/openproblems-bio/openproblems/blob/main/CONTRIBUTING.md
  - text: OpenProblems v2
    href: https://github.com/openproblems-bio/openproblems-v2/blob/main/CONTRIBUTING.md
```

```yaml
menu:
  - text: OpenProblems v1
    href: https://github.com/openproblems-bio/openproblems/blob/main/CONTRIBUTING.md
    target: _blank
  - text: OpenProblems v2
    href: https://github.com/openproblems-bio/openproblems-v2/blob/main/CONTRIBUTING.md
    target: _blank
```

### LINKED ISSUE #149: documentation links in navbar open in same tab
when quarto 1.3 releases add `target: _blank` in navbar option for documentation in the `_quarto.yaml` file. If documentation for v2 is not finished. see quarto-dev/quarto-cli#2284

```yaml
menu:
  - text: OpenProblems v1
    href: https://github.com/openproblems-bio/openproblems/blob/main/CONTRIBUTING.md
  - text: OpenProblems v2
    href: https://github.com/openproblems-bio/openproblems-v2/blob/main/CONTRIBUTING.md
```

```yaml
menu:
  - text: OpenProblems v1
    href: https://github.com/openproblems-bio/openproblems/blob/main/CONTRIBUTING.md
    target: _blank
  - text: OpenProblems v2
    href: https://github.com/openproblems-bio/openproblems-v2/blob/main/CONTRIBUTING.md
    target: _blank
```

### PR DESCRIPTION
**Pull request by scottgigante-immunai**
Closes openproblems-bio/website#149

### CODE REVIEW

File: _quarto.yml [hunk 1]
```diff
@@ -36,8 +36,10 @@ website:
         menu:
           - text: OpenProblems v1
             href: https://github.com/openproblems-bio/openproblems/blob/main/CONTRIBUTING.md
+            link-external-newwindow: true
```
Comment (KaiWaldrant) [inline _quarto.yml:39]: `link-external-newwindow: true` also doesn't work. This cannot be used that's why a specific issue was added to quarto see my issue. this is fixed in quarto 1.3

**Review by KaiWaldrant — CHANGES_REQUESTED** [review]
I think we can just remove this PR ?

File: _quarto.yml (hunk 1 above)
Comment (scottgigante-immunai) [reply _quarto.yml]: Ah, the problem is our quarto version? Why don't we upgrade?

Comment (KaiWaldrant) [reply _quarto.yml]: 1.3 is in prerelease

**Pull request closed without merging**